app/src/app/decret-form/decret-form.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { ALL_DECREE } from '../../../../common/config';
+import { AngularFireDatabase, snapshotChanges } from '@angular/fire/database';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { MatDialog } from '@angular/material';
+import { take, tap } from 'rxjs/operators';
+import { AvailableQuestion } from '../model';
+
+@Component({
+  selector: 'app-decret-form',
+  templateUrl: './decret-form.component.html',
+  styleUrls: ['./decret-form.component.css']
+})
+export class DecretFormComponent implements OnInit {
+
+  constructor(private fb: FormBuilder, private db: AngularFireDatabase, private auth: AngularFireAuth, private dialog: MatDialog) { }
+
+  decretForm: FormGroup;
+  title: string
+  state: string;
+
+  allDecree = ALL_DECREE
+  availableDecree: String[]
+
+  @Input()
+  house: AvailableQuestion
+
+  @Input()
+  roundId: string
+  
+  @Input()
+  delegateId: string
+
+  questionsPath: string
+
+  ngOnInit() {
+    this.decretForm = this.fb.group({
+      decretType: [""],
```

Comment:
`decree` vs `decret` naming should be unified

Reply:
Isn't decree plural vs decret singular? :)

Reply:
I was using decree when I wanted to use plural.

Reply:
From what I found 'décret' is a French word and it's not used much in English. We should use English word decree. Plural is decrees.

Reply:
Ok, will rename.

Reply:
I had idea today - how about leaving decrees completely and stay with questions? It's saved into questions to db so it makes sense. So here it will be questionType

Reply:
Yup that's no prob, I will be unifying the terminology now, so thats fine. But in the texts visible to users we should use "Dekret" :)

Reply:
Terminology changed, except for the `decret-form-component`. As that would be really cumbersome to remake.